code/modules/food_and_drinks/food/snacks_cake.dm

```diff
@@ -254,5 +244,90 @@
 	icon_state = "pumpkinspicecakeslice"
 	filling_color = "#FFD700"
 	tastes = list("cake" = 5, "sweetness" = 1, "pumpkin" = 1)
-
 	foodtype = GRAIN | DAIRY | VEGETABLES | SUGAR
+
+/obj/item/reagent_containers/food/snacks/store/cake/bsvc // blackberry strawberries vanilla cake
+	name = "blackberry and strawberry vanilla cake"
+	desc = "A plain cake, filled with assortment of blackberries and strawberries!"
+	icon_state = "blackbarry_strawberries_cake_vanilla_cake"
+	slice_path = /obj/item/reagent_containers/food/snacks/cakeslice/bsvc
+	bonus_reagents = list(/datum/reagent/consumable/nutriment = 14, /datum/reagent/consumable/nutriment/vitamin = 4)
+	tastes = list("blackberry" = 2, "strawberries" = 2, "vanilla" = 2, "sweetness" = 2, "cake" = 3)
+	foodtype = GRAIN | DAIRY | FRUIT | SUGAR
+
+/obj/item/reagent_containers/food/snacks/cakeslice/bsvc
+	name = "blackberry and strawberry vanilla cake slice"
+	desc = "Just a slice of cake  filled with assortment of blackberries and strawberries!"
+	icon_state = "blackbarry_strawberries_cake_vanilla_slice"
+	filling_color = "#FFD700"
+	tastes = list("blackberry" = 2, "strawberries" = 2, "vanilla" = 2, "sweetness" = 2,"cake" = 3)
+	foodtype = GRAIN | DAIRY | FRUIT | SUGAR
+
+/obj/item/reagent_containers/food/snacks/store/cake/bscc // blackbarry strawberries chocolate cake
+	name = "blackberry and strawberry chocolate cake"
+	desc = "A plain cake, filled with assortment of blackberries and strawberries!"
```

Comment:
Should this be "A chocolate cake, filled with..." rather than plain?

Reply:
! Yes o7
Thanks for pointing that out. Coding well tired not the best idea! >_<